src/pages/register-a-beacon/check-your-answers.tsx

```diff
@@ -101,6 +108,20 @@ export const getServerSideProps: GetServerSideProps = async (
   context: GetServerSidePropsContext
 ) => {
   // TODO: State persistence stuff to go here
+  if (context.req.method === "POST") {
+    await axios
+      .post(`${serverRuntimeConfig.apiUrl}/beacons`, { beaconStatus: "idk" })
```

Comment:
@Ninamma LGTM! 👍  Nice work.

To check if this works without the `.env.local` file I would try removing it and setting the environment variable, so something like `export API_URL="http://localhost:8080/api` in your local terminal and then restart the app.  This should still work if the API url has been passed in properly.  What that will mean is that we will be able to set this variable within the [integration](https://github.com/madetech/mca-beacons-integration) project so allow the registration app to talk to the API service!